Config/Inc/config.h

```diff
@@ -54,4 +54,9 @@ typedef enum {SAFE = 0, DANGER = 1, OVERVOLTAGE = 2, UNDERVOLTAGE = 3} SafetySta
 // Define how many temperature sensors are connected to each board
 #define MAX_TEMP_SENSORS_PER_MINION_BOARD	16
 
+//--------------------------------------------------------------------------------
+// HeartBeat Delay Ticks
+// Define heartbeatDelay as # of desired while(1) loops per toggle
+#define heartbeatDelay 0x6000
```

Comment:
Can you make this all caps? Generally we make #defines all caps and variables not all caps, so it is easier to tell them apart at a glance.

Reply:
Additionally, this value should be much smaller for the run light to toggle in a reasonable amount of time (I got this working with 0x60, but you can experiment to find a value that you think is best)

Reply:
Out of curiosity, does the simulator run at real-time, or is it faster than what we would expect on the board?